src/main/scala/com/typesafe/sbt/packager/jdkpackager/JDKPackagerHelper.scala

```diff
@@ -11,19 +15,45 @@ import sbt._
 object JDKPackagerHelper {
 
   /** Attempts to compute the path to the `javapackager` tool. */
-  def locateJDKPackagerTool(): Option[File] = {
-    val jdkHome = sys.props.get("java.home").map(p ⇒ file(p))
-
-    // TODO: How to get version of javaHome target?
-    val javaVersion = VersionNumber(sys.props("java.specification.version"))
-    val toolname = javaVersion.numbers.take(2) match {
-      case Seq(1, m) if m >= 8 ⇒ "javapackager"
-      case _                   ⇒ sys.error("Need at least JDK version 1.8 to run JDKPackager plugin")
+  def locateJDKPackagerTool(javaHome: Option[File]): Option[File] = {
+    val toolname = sys.props("os.name").toLowerCase match {
+      case os if os.contains("win") ⇒ "javapackager.exe"
+      case _                        ⇒ "javapackager"
     }
-    jdkHome
-      .map(f ⇒ if (f.getName == "jre") f / ".." else f)
-      .map(f ⇒ f / "bin" / toolname)
-      .filter(_.exists())
+
+    // This approach to getting JDK bits is borrowed from: http://stackoverflow.com/a/25163628/296509
+    // Starting with an ordered list of possible java directory sources, create derivative and
+    // then test for the tool. It's nasty looking because there's no canonical way of finding the
+    // JDK from the JRE, and JDK_HOME isn't always defined.
+    Seq(
+      // Build-defined
+      javaHome,
```

Comment:
Nice, I didn't know that setting. One couldn't believe how hard it is to find the JDK on a system -.-